client/src/components/EmailFrontend.js

```diff
@@ -50,12 +50,12 @@ const EnumField = ({
   selected_enums_for_field,
   disabled,
   state_update_callback,
-  classes,
 }) => (
-  <div className={`${form_type} ${classes}`}>
+  <div className={`${form_type}`}>
     <label htmlFor={`${field_id}--${enum_key}`}>
       <input
         id={`${field_id}--${enum_key}`}
+        style={{ marginRight: "1rem" }}
```

Comment:
Saw you using rem's, here and elsewhere. I'd not recommend it. rem and em are good if you're careful and consistent in their use, or if they're being used for specific layouts that really should scale with font size changes. I don't think we want some obscure margins shifting around if we ever change the font on the root element though.